docs/docs/.vuepress/config.js

```diff
@@ -52,35 +70,52 @@ module.exports = {
         ],
       },
       {
-        title: "Workflows",
+        title: "Code",
```

Comment:
In the hierarchy of concepts, do you think Workflows should go above Code instead? That way the order of the concepts in the sidebar go: Overview -> Workflows -> things you can do in workflows (like code, connect accounts, etc.)

Reply:
Yup 100% agreed. The user journey should start with Workflows before Code.